docs/source/validation.rst

```diff
@@ -0,0 +1,461 @@
+
+            geomref = OBSColumn(
+              type='Text',
+              targets={geom: GEOM_REF})
+
+            return OrderedDict([
+              ('geom', geom),
+              ('geomref', geomref)
+            ])
+
+    class MyColumnsTask(ColumnsTask):
+
+        def columns(self):
+            # Return OrderdDict of columns here
+
+    class MyTableTask(TableTask):
+
+        def timespan(self):
+            # Return timespan here
+
+        def requires(self):
+            return {
+                'geom_columns': MyGeoColumnsTask(),
+                'data_columns': MyColumnsTask()
+             }
+
+        def columns(self):
+            cols = OrderedDict()
+            cols['geomref'] = self.input()['geom_columns']['geomref']
+            cols.update(self.input()['data_columns'])
+            return cols
+
+        def populate(self):
+            # Populate the output table here
+
+The above code would ensure that all columns existing inside ``MyTableTask``
+would be appropriately linked to any geometries that connect to ``geomref``.
+
+Do you have both the data and geometries in your table?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can check by running:
+
+.. code:: sql
+
+    SELECT * FROM observatory.obs_table
+    WHERE id LIKE 'my.schema.%';
+
+If there is only one table and it has a null "the_geom" boundary,
+then you are missing a geometry table. For example:
+
+.. code:: sql
+
+   SELECT * from observatory.obs_table
+   WHERE id LIKE 'es.ine.five_year_population%';
+
+.. code:: shell
+
+                      id                   |                  tablename                   | timespan | the_geom | description | version 
+   ----------------------------------------+----------------------------------------------+----------+----------+-------------+---------
+    es.ine.five_year_population_99914b932b | obs_24b656e9e23d1dac2c8ab5786a388f9bf0f4e5ae | 2015     |          |             |       5
+   (1 row)
+
+Notice that the_geom is empty. You will need to write a second :ref:`TableTask` with the 
+following structure:
+
+.. code:: python
+
+   class Geometry(TableTask):
+
+        def timespan(self):
+            # Return timespan here
+
+        def requires(self):
+           return {
+               'meta': MyGeoColumnsTask(),
+               'data': RawGeometry()
+           }
+           
+       def columns(self):
+           return self.input()['meta']
+
+        def populate(self):
+            # Populate the output table here
+
+Regenerate and look at the Catalog
+**********************************
+
+Once :ref:`tasks.carto.OBSMetaToLocal` has been run, you can generate the
+catalog.
+
+.. code:: shell
+
+     make catalog
+
+1. Are there any nasty typos or missing data?
+2. Does the nesting look right?  Are there columns not nested?
+3. Are sources and licenses populated for all measures?
+4. Is a table with a boundary/timespan matrix appearing beneath each measure?
+
```

Comment:
1. Are there any nasty typos or missing data?
     - Variable names should be unique, human-readable, and concise. If the variable needs more in-depth definition, this should go in the "description" of the variable.
2. Does the nesting look right? Are there columns not nested?
     - Variables that are denominators should also have subcolumns of direct nested variables.
     - There may be repetitive nesting if a variable is nested under two denominators, which is fine.
3. Are sources and licenses populated for all measures?
     - A ``source`` and ``license`` :ref:`tasks.util.OBSTag` must be written for new sources and licenses
4. Is a table with a boundary/timespan matrix appearing beneath each measure?
5. Are code blocks generated with a sample latitude and longitude for the area?
     - If not, hardcode the sample latitude and longitude in :ref:`tasks.meta.catalog_lonlat`